src/arfsdao.ts

```diff
@@ -491,7 +736,7 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 		return transaction;
 	}
 
-	async getPrivateDrive(driveId: string): Promise<ArFSPrivateDrive> {
+	async getPrivateDrive(driveId: DriveID): Promise<ArFSPrivateDrive> {
```

Comment:
Running this command for private folders (after fixing the below GQL tag issue) will result in:

```
(node:23977) UnhandledPromiseRejectionWarning: Error: Invalid folder state
```

Private folders and drives will require the drivePassword or driveKey at this layer in order to decrypt the `name` and `parent-folder-id` tags from the data json. 

I actually made a change coming up in the `cli_v2_smartweave` branch to the `drive-info` command in order to get it to work for Private Drives: https://github.com/ardriveapp/ardrive-cli/commit/9e2118d38f05dd07b5242f37bd3f7cdaadbe41b3

Reply:
Merged your work from cli_v2: 
68e610a